recipes/soprano/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set name = "Soprano" %}
+{% set version = "0.8.13b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CCP-NC/soprano/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e1d8ab62c4d410ab1052021d7ae645296fec2bb8d63938014584c6aa7038961b
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - vasp2cell = soprano.scripts.vasp2cell:__main__
+    - soprano_submitter = soprano.hpc.submitter:__main__
+    - phylogen = soprano.scripts.phylogen:__main__
+    - magresaverage = soprano.scripts.msaverage:__main__
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - scipy
+    - ase
+
+test:
+  source_files:
+    - tests/
```

I think I made a mistake earlier. Try removing the trailing `/`?
```suggestion
    - tests
```